clients/v1_auth/v1_auth.go

```diff
@@ -0,0 +1,428 @@
+	newAuth, err := c.AuthorizationsApi.PostAuthorizations(ctx).AuthorizationPostRequest(authReq).Execute()
+	if err != nil {
+		return fmt.Errorf("failed to create new authorization: %w", err)
+	}
+
+	if password != "" {
+		if err := c.UsersApi.PostUsersIDPassword(ctx, newAuth.GetUserID()).
+			PasswordResetBody(api.PasswordResetBody{
+				Password: password,
+			}).Execute(); err != nil {
+			_ = c.AuthorizationsApi.DeleteAuthorizationsID(ctx, newAuth.GetId()).Execute()
+			return fmt.Errorf("failed to update password for user ID %q: %w", newAuth.GetUserID(), err)
+		}
+	}
+
+	usr, err := c.UsersApi.GetUsersID(ctx, newAuth.GetUserID()).Execute()
+	if err != nil {
+		return err
+	}
+
+	ps := make([]string, 0, len(auth.Permissions))
+	for _, p := range auth.Permissions {
+		ps = append(ps, permString(p))
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{
+		token: &v1Token{
+			ID:          newAuth.GetId(),
+			Description: newAuth.GetDescription(),
+			Token:       newAuth.GetToken(),
+			Status:      newAuth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type RemoveParams struct {
+	clients.AuthLookupParams
+}
+
+func (c Client) Remove(ctx context.Context, params *RemoveParams) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	auth, err1 := c.AuthorizationsApi.GetAuthorizationsID(ctx, id).Execute()
+	err2 := c.AuthorizationsApi.DeleteAuthorizationsID(ctx, id).Execute()
+	if err1 != nil || err2 != nil {
+		return fmt.Errorf("could not find Authorization from ID %q: %w", id, err1)
+	}
+
+	usr, err := c.UsersApi.GetUsersID(ctx, auth.GetUserID()).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find user from user ID %q: %w", auth.GetUserID(), err)
+	}
+
+	ps := make([]string, 0, len(auth.GetPermissions()))
+	for _, p := range auth.GetPermissions() {
+		ps = append(ps, permString(p))
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{
+		deleted: true,
+		token: &v1Token{
+			ID:          auth.GetId(),
+			Description: auth.GetDescription(),
+			Token:       auth.GetToken(),
+			Status:      auth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type ListParams struct {
+	clients.OrgParams
+	clients.AuthLookupParams
+	User   string
+	UserID string
+}
+
+func (c Client) List(ctx context.Context, params *ListParams) error {
+	req := c.AuthorizationsApi.GetAuthorizations(ctx)
+
+	if params.User != "" {
+		req = req.User(params.User)
+	}
+	if params.OrgID.Valid() {
+		req = req.OrgID(params.OrgID.String())
+	}
+	if params.OrgName != "" {
+		req = req.Org(params.OrgName)
+	}
+	var id string
+	if params.AuthLookupParams.IsSet() {
+		newId, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+		if err != nil {
+			return fmt.Errorf("could not get User from Username %q: %w", params.Username, err)
+		}
+		id = newId
+	}
+	if params.UserID != "" {
+		id = params.UserID
```

Comment:
I don't think this is correct. `params.UserID` is the ID of the v2 user that owns the token, so it should get set directly on the request as `req.UserID`.